x-pack/plugin/watcher/qa/rest/src/test/resources/rest-api-spec/test/mustache/30_search_input.yml

```diff
@@ -13,6 +16,8 @@ setup:
             "date" : "2015-01-01T00:00:00",
             "value" : "val_1"
           }
+      warnings:
+        - "[xpack.ilm.enabled] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version."
```

Comment:
why are we getting these warnings? Since this is testing ilm enabled, shoudln't the test setup be using the default and thus we get no warning? I think the problem is many of these qa tests are trying to disable other features, but we should simply remove those settings from build.gradle in the cluster setup?

Reply:
I agree. I've been trying to identify the specific places where deprecation warnings are thrown in order to make sure that there's nothing important to the tests about having particular settings in `build.gradle`, but in the case of the watcher tests, the deprecation warnings seem to be happening on index creations and deletions, which doesn't seem relevant to watcher.